### PR TITLE
Add recipe for r-text2vec

### PR DESCRIPTION
**Pull request by CurtLH**
Recipe created with [conda-r-forge-skeleton-helper](https://github.com/bgruening/conda_r_skeleton_helper)

### CODE REVIEW

**Comment by CurtLH** [comment]
Once the dependencies for `r-text2vec` (PR #5566, PR #5567) are merged, I will push my branch again.

File: recipes/r-text2vec/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    - r-stringi >=1.1.5
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('text2vec')"  # [not win]
+    - "\"%R%\" -e \"library('text2vec')\""  # [win]
+
+about:
+  home: http
```
Comment (cbrueffer) [inline recipes/r-text2vec/meta.yaml]: Could add a homepage? The "url" from the CRAN page is usually good.

File: recipes/r-text2vec/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  license: GPL (>= 2)
+  summary: Fast and memory-friendly tools for text vectorization, topic modeling (LDA, LSA),
+    word embeddings (GloVe), similarities. This package provides a source-agnostic streaming
+    API, which allows researchers to perform analysis of collections of documents which
+    are larger than available RAM. All core functions are parallelized to benefit from
+    multicore machines.
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
```
Comment (cbrueffer) [inline recipes/r-text2vec/meta.yaml]: Duplicated licenses.

**Comment by CurtLH** [comment]
@cbrueffer - now that the dependencies have been merged (PR #5566, PR #5567), this should be ready to merge as well.

**Comment by CurtLH** [comment]
Any what what went wrong here? I've been reviewing the build log, but I'm not familiar enough with the process to understand where the problem is.

File: recipes/r-text2vec/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-matrix >=1.1
+    - r-r6 >=2.1.2
+    - r-rcpp >=0.11
+    - r-rcppparallel >=4.3.14  # [win]
```
Comment (cbrueffer) [inline recipes/r-text2vec/meta.yaml]: Looks like this should be an unconditional dependency?

**Comment by CurtLH** [comment]
@cbrueffer -- I removed the conditional dependency for `r-rcppparallel`, but the AppVeyor Build and Travis CI build are still both failing.  

From searching in [Anaconda Cloud](https://anaconda.org/conda-forge/r-rcppparallel/), it looks like `r-rcppparallel` only builds for `linux-64` and `osx-64`, so I think I need to just skip Windows all together.  The changes I made seem to get me closer, but I'm still missing something.

**Comment by CurtLH** [comment]
Thanks for the information @cbrueffer.  Given the situation, and the fact that I don't see any development happening on [`RcppParallel`](https://github.com/RcppCore/RcppParallel), can we just build this package for Linux now and revisit when the OSX is resolved? Would that be an acceptable approach for now?

**Comment by cbrueffer** [comment]
Seems OK to me; please document why osx and win64 are excluded in a comment above "skip" so we can track this.

File: recipes/r-text2vec/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  url:
+    - https://cran.r-project.org/src/contrib/text2vec_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/text2vec/text2vec_{{ version }}.tar.gz
+  sha256: ecf97ef61578f69d5cb6691385a01ed10080aa2914b54d4671abbffb2a28976d
+
+build:
+  number: 0
+  # skip osx and win builds because of issue with r-rcppparallel
+  # the OSX error seems to be related to C++11
+  skip: true  # [osx and win]
```
Comment (cbrueffer) [inline recipes/r-text2vec/meta.yaml]: This should be "or".

File: recipes/r-text2vec/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  url:
+    - https://cran.r-project.org/src/contrib/text2vec_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/text2vec/text2vec_{{ version }}.tar.gz
+  sha256: ecf97ef61578f69d5cb6691385a01ed10080aa2914b54d4671abbffb2a28976d
+
+build:
+  number: 0
+  # skip osx and win builds because of issue with r-rcppparallel
+  # the OSX error seems to be related to C++11
+  skip: true  # [osx or win]
```
Comment (cbrueffer) [inline recipes/r-text2vec/meta.yaml]: Maybe spelling this as "True" fixes the error, although I would have expected the LINT checks to complain of it was a problem.

**Comment by cbrueffer** [comment]
The recipe looks find, not sure why it's failing. Looks like there have been updates to the build infrastructure, you could try rebasing your branch to the latest staged-recipes to up those updates.

**Comment by CurtLH** [comment]
It looks like this is good to merge @cbrueffer.  Thank you for all of your help with this one.

**Pull request merged**